virtual_rainforest/models/animals/__init__.py

```diff
@@ -1,15 +1,15 @@
-"""The :mod:`~virtual_rainforest.models.animal` module is one of the component models
+"""The :mod:`~virtual_rainforest.models.animals` module is one of the component models
 of the Virtual Rainforest. It is comprised of a number of submodules.
 
 Each of the animal sub-modules has its own API reference page:
 
-* The :mod:`~virtual_rainforest.models.soil.animal_model` submodule instantiates the
+* The :mod:`~virtual_rainforest.models.animals.animal_model` submodule instantiates the
```

Comment:
You probably should mention the following api pages in here: animal_communities.md, animal_cohorts.md, functional_group.md, scaling_functions.md, dummy_plants_and_soil.md

Reply:
Thanks for the reminder! They were on the list and then I got very wrapped up in the weird errors I was getting.

Reply:
Yes `sphinx` errors are an all consuming nightmare